src/Debugger.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Yiisoft\Yii\Debug;
+
+use Yiisoft\Yii\Debug\Target\TargetInterface;
+
+class Debugger
+{
+    private string $id;
+    /**
+     * @var \Yiisoft\Yii\Debug\Collector\CollectorInterface[]
+     */
+    private array $collectors;
+    private TargetInterface $target;
+
+    public function __construct(TargetInterface $target, array $collectors)
+    {
+        $this->collectors = $collectors;
+        $this->target = $target;
+        $this->id = uniqid('yii-debug-', true);
+    }
+
+    public function getId(): string
```

Comment:
What do we need ID for?

Reply:
It provides `x-debug-tag` like in yii2. It used to view result after a page was processed